Makefile

```diff
@@ -381,7 +381,12 @@ istioctl-install:
 .PHONY: test localTestEnv test-bins
 
 # Run coverage tests
-test: pilot-test mixer-test security-test broker-test galley-test common-test
+JUNIT_UNIT_TEST_XML ?= $(ISTIO_OUT)/junit_unit_tests.xml
```

Comment:
Please also update .circleci to use this target if possible

Reply:
It is the same `test` target that circle ci has been using. By updating the target implementation, it always generates the junit results regardless of caller.